src/components/Typography/index.js

```diff
@@ -1,7 +1,7 @@
-export { default as H1 } from './h1'
-export { default as H2 } from './h2'
-export { default as H3 } from './h3'
-export { default as H4 } from './h4'
-export { default as H5 } from './h5'
-export { default as H6 } from './h6'
-export { default as Paragraph } from './p'
+export { default as SectionTitleH1 } from './SectionTitleH1'
```

Comment:
Might be my lack of food, but could a possible for solution for naming be the following:

- We only have one H1, so it should be `<H1>`
- We'd have one `<H2>` with a modifier for `display` (it would fallback for section)
- We'd have one `<H3>` with a modifier for `card` (it would fallback  for the subtitle)
- The body ones would stay the same.

I'm not so close to the designer so I might be missing something, if so, don't mind me!

Reply:
I think one of the desired effects of this PR is to separate styles from heading hierarchy as far as possible.

We have to give each of these sets of styles a default heading number. So I've gone with the one that these styles are most commonly applied to. For example, we usually use the Subtitle styles on an HTML element that happens to be `h3` in the hierarchy.

For all other situations, we can use the `as="hX"` prop.

In our situation, we rarely actually use an h1 anywhere on the page, because this is reserved for SEO & is therefore only used on:
- the HidddenText element (which is not visible to a user and therefore needs no styles attached)
- the `SEOText` component

So, arguably, we don't actually need `SectionTitleH1` at all.

Does that makes sense? Happy to have a call about it tomorrow with all designers & engineers, to clarify the situation.

Reply:
resolved via a call between all engineers